### ISSUE TITLE
Lithium Subproject activities plan

### ISSUE THREAD

**Issue by Prokster** [uncefact/project-crm#7]

**Comment by onthebreeze** [comment]
yep - outline is here https://uncefact.github.io/project-crm/docs/pilots/Pilot-AUS and content to edit is here https://github.com/uncefact/project-crm/blob/main/website/docs/pilots/Pilot-AUS.md.  Since you are sub-project lead, lets get you up to speed on github (if not already).
